NetKAN/AviationLights.netkan

```diff
@@ -2,25 +2,17 @@
     "spec_version": "v1.4",
     "identifier":   "AviationLights",
     "name":         "Aviation Lights",
-    "author":       [ "MOARdV", "BigNose", "RPGprayer" ],
-    "$kref":        "#/ckan/github/MOARdV/AviationLights",
+    "author":       [ "LisiasT", "MOARdV", "BigNose", "RPGprayer" ],
+    "$kref":        "#/ckan/github/spacedock/3208",
```

Comment:
```suggestion
    "$kref":        "#/ckan/spacedock/3208",
```

Reply:
UGH… I just fixed it on a commit below...